feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/stats_nl80211.c

```diff
@@ -471,17 +472,17 @@ static struct nl_msg *nl80211_call_vif(struct nl_call_param *nl_call_param, int
 	struct nl_msg *msg;
 
 	if (!nl80211_scan_started) {
```

Comment:
I've always wondered about this block of code.  It looks to me like it could be done at start-up.

Reply:
I agree. 
Logically, it does not make any difference as we are gating the initialization with "nl80211_scan_started". But yes, we can move this block to init and get rid of "nl80211_scan_started".
I will make this change unless anyone objects to it.

Reply:
Done